contracts/coordinator/src/events.rs

```diff
@@ -0,0 +1,9 @@
+use axelar_wasm_std::IntoEvent;
+use cosmwasm_std::Addr;
+
+#[derive(IntoEvent)]
+pub enum ContractInstantiated {
+    Gateway { address: Addr, code_id: u64 },
+    VotingVerifier { address: Addr, code_id: u64 },
+    MultisigProver { address: Addr, code_id: u64 },
+}
```

Comment:
should this include the deployment_name as well? could this also be arranged something like this:
```suggestion
pub struct ContractInstantiated {
    address: Addr,
    code_id: u64,
    deployment_name: String,
    contract: Contract, // enum
}
```

Reply:
Will resolve in another PR